packages/pds/tests/views/notifications.test.ts

```diff
@@ -70,13 +70,16 @@ describe('pds notification views', () => {
   })
 
   it('paginates', async () => {
+    // @TODO there is a small bug where notifications generated with the same
```

Comment:
yeah :/ this is actually an issue with all our pagination i believe :/

Reply:
I think you're right.  The exception is the actor search, whose cursor is a pair `(score, did)`.